code/modules/atmospherics/machinery/components/fusion/hfr_core.dm

```diff
@@ -159,6 +159,17 @@
 	///Var used in the meltdown phase
 	var/final_countdown = FALSE
 
+	///Damage caused because of the high power tier
+	var/high_power_damage = FALSE
+	///Damage caused by too much fuel gas inside
+	var/high_fuel_mix_mole = FALSE
+	///Damage caused by iron content
+	var/iron_content_damage = FALSE
+	///Check if iron inside the core is increasing
+	var/iron_content_increasing = FALSE
+	///If this is true only a non readable message will pop out
+	var/emped = FALSE
```

Comment:
You should turn these into bitflags, I get the feeling it'll make the code slightly nicer